daily-challenges/2024-09/14/hunghoang3110.spec.ts

```
@@ -0,0 +1,41 @@
+        phanloai = 'Beo phi';
+    }
+    console.log(`Ket qua BMI: ${bmi}  \nPhan loai : ${phanloai}`);
+}
+
+caculateBMI(60,1.68);
+
+// Playwright
+import {test, expect} from '@playwright/test';
+
+test('check username and email', async ({page}) => {
+    var user_name = 'carter.nguyen', email_name = 'carter.nguyen@gmail.com';
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)